feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -137,8 +137,21 @@ fun PreviewScreen(
             aspectRatio = previewUiState.currentCameraSettings.aspectRatio,
             deferredSurfaceProvider = deferredSurfaceProvider
         )
-        // overlay
-        Box(
+        // quick settings screen overlay
+        QuickSettingsScreen(
```

Comment:
nit: We should name this QuickSettinsgScreenOverlay, it's better for the names to be self documenting.